content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,173 @@
+# Merge Conflicts
```

Comment:
Resolving Merge Conflicts ? (depends on how to we call other sections)

Reply:
I think a shorter title is better in general. As long as we have the keyword "resolve" throughout the doc, it should be fine.